deliverydb/db.py

```diff
@@ -40,3 +37,21 @@ def insert_compliance_issue(
         )
 
         self.Session.add(scan)
+
+
+def database_conncetion_url_from_cfg(
+    db_cfg: model.compliancedb.ComplianceDbConfig,
+    dialect: str = 'postgresql',
+    overwrite_hostname: str = None,
+    overwrite_username: str = None,
+    overwrite_password: str = None,
+    overwrite_port: int = None,
+) -> str:
+    username = overwrite_username if overwrite_username else db_cfg.credentials().username()
```

Comment:
stylistic nit: you might use `username = overwrite_username or db_cfg.credentials().username()` as a slightly shorter style